lib/profile/sections/sections.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:flutter/material.dart';
+import 'package:mentorApp/constants/navigation_constants.dart';
+import 'package:mentorApp/constants/profile_constants.dart';
+import './summary.dart';
+import './education.dart';
+import './experience.dart';
+import './skills.dart';
+import './contact.dart';
+
+ 
+class ProfileSections extends StatelessWidget {  
+
+  final Map data;
+  final Map<String, GlobalKey> keys;
+  ProfileSections({Key key, @required this.data, @required this.keys}) : super(key: key);
+  
+  @override
+  Widget build(BuildContext context) {
+
+    return 
+    Container(
+      height: MediaQuery.of(context).size.height - kProfileHeaderHeight - kAppNavBarHeight,
```

Comment:
Here is why I assigned a definite heights for profile header and app nav bar.

Reply:
this makes sense for now but in a later PR we might want this to be done using different flex factors or something like that

Reply:
Agreed. Will make a note of this!